src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
```

Comment:
The optional chaining operator on 'people' is unnecessary here since 'people' is a required array prop and cannot be undefined.